code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -96,7 +96,7 @@
 				jammed = TRUE
 			jamming_chance  += jamming_increment
 			jamming_chance = clamp (jamming_chance, 0, 100)
-	..()
+	. = ..()
```

Comment:
```suggestion
	return ..()
```

Reply:
What's the difference between these? I always thought they meant the same thing. But I know now one calls the parent proc on the line it's on and the other returns early and calls the parent proc. I don't know what the . means.

Reply:
They mean the same thing in this specific case, it's just `return` at the end is more clear.

Reply:
ah, alright

Reply:
I thought return ..() stopped the rest of the code running while . = ..() just ran it after that line